CuratedContent/SoftwareSustainabilityMatrix.md

```diff
@@ -0,0 +1,81 @@
+The goal of the SSM effort is to create numerical scores for different aspects of software sustainability for a software package as well of a combined sustainability score.
+These SSM sustainability scores are being to guide efforts to improve the sustainability of the software package ecosystem making to CIBC software portfolio, with the hopes that it will encourage support and contributions from the open-source community and reduce the cost and risks to sustain this collection software.
+However, the SSM approach may be applicable to other package ecosystems and therefore may be of more general interest.
+
+The SSM score is composed from component scores in the four areas Impact, Risks, Community, and Technology as shown in [Figure 1](#fig_software_sustainability_matrix) and each of these areas are broken down into several different subareas.
+
+<a name="fig_software_sustainability_matrix"/>
+
+<br>
+<center>
+<img src='../images/SoftwareSustainabilityMatrix.png' class='page'/>
+</center>
+<br>
+
+<center>
+<b>Figure 1:</b> Software Sustainability Matrix
+</center>
+
+<br>
+<br>
+ 
+Scores in the range [0-100] in these four areas Impact, Risks, Community, and Technology are then combined in a weighted sum:
+
+&nbsp;&nbsp;&nbsp;&nbsp;SSM Score = F<sup>I</sup> * I + F<sup>R</sup> * R + F<sup>C</sup> * C + F<sup>T</sup> * T
+
+to produce a single SSM Score [0-100], where F<sup>I</sup> + F<sup>R</sup> + F<sup>C</sup> + F<sup>T</sup> = 1.
+The SCI & Kitware process<sup>[3]</sup> typically uses the weighting factors F<sup>I</sup> = F<sup>C</sup> = 1/3, and F<sup>R</sup> = F<sup>T</sup> = 1/6.
+It is not specified how the SCI & Ktiware program computes the component scores in the four areas 'I', 'R', 'C', and 'T' for the software packages of interest and the authors state that computing these individual component scores is ultimately a subjective matter.
+
+It is stated by the authors that the primary benefit of estimating the SSM score by a software package team is not in computing the final score, but instead in highlighting areas of possible improvement.
+It is stated that many teams make significant improvements in the sustainability for their packages while working through in the process of estimating the SSM score by adopting better processes in different areas.
+In this respect, this is similar to the impact of the OpenSSF Best Practices Badge Program<sup>[4]</sup>.
+That is, by having a team fill out each best practice item, the team will often implement suggested best practices, especially for practices that are not too difficult to adopt and support.
+
+In summary, the Software Sustainability Matrix (SSM) is a promising approach to help quantify the sustainability of individual software packages (and collectively entire software package ecosystems) and then help to drive efforts to make targeted improvements to improve sustainability.
```

Comment:
```suggestion
In summary, the Software Sustainability Matrix (SSM) is a promising approach to help quantify the sustainability of individual software packages (and collectively entire software package ecosystems) and subsequently help to drive efforts to make targeted improvements for sustainability.
```

Reply:
I applied the suggested change in commit e325da88e801036e9a5daf42009a94239c9f5437.